pmm-tests/pmm-framework.sh

```diff
@@ -1841,9 +1860,9 @@ if [ ! -z $setup ]; then
   setup
 fi
 
-if [ ! -z $PMM2 ]; then
-  configure_client;
-fi
+#if [ ! -z $PMM2 ]; then
```

Comment:
Why comment on this?

Reply:
I removed it, because of client's configuration is called in installation of pmm2-client (see 700 line) and I think we don't need to call it each time. I checked installation of PMM server and adding clients and everything worked ok without this part